bbn-test-5/babylon-validators/README.md

```diff
@@ -0,0 +1,358 @@
+
+Upon successful creation, you'll be asked to approve the transaction. 
+Once approved, you'll receive a transaction hash and your validator
+operator address 
+(e.g., `bbnvaloper1qh8444k43spt6m8ernm8phxr332k85teavxmuq`).
+
+### 5.1 Verifying Validator Setup
+
+To verify your validator setup, you can use the following steps:
+
+First, get your validator's operator address using your Babylon address: 
+
+```shell
+babylond keys show <your-key-name> --address --bech val --home <path> --keyring-backend <keyring-backend>
+```
+
+For example, for the address we used above is `bbn1qh8444k43spt6m8ernm8phxr332k85teavxmuq`, 
+the operator address is `bbnvaloper1qh8444k43spt6m8ernm8phxr332k85teavxmuq`. 
+
+For the next step, we will have to wait till the beginning of the next epoch 
+(this will take up to 10 minutes) for the command to be valid. This is due to 
+Babylon's epoched staking design, where the blockchain is divided into epochs.
+Messages that affect the validator set are delayed to the end of each epoch for 
+execution. Meaning that your validator creation transaction will be processed 
+at the end of the current epoch, and you won't be able to query your validator 
+details until the next epoch begins. In the meantime, you can verify your 
+transaction was successful by checking its hash.
+
+```shell 
+babylond query staking validator <validator-operator-address>
+```
+
+The output should return the selected validator's configuration.
+
+```yaml 
+validator:
+  commission:
+    rates:
+      current: "100000000000000000" 
+      max: "1000000000000000000" 
+      max_change: "10000000000000000"
+  description:
+    moniker: "my-validator" 
+    website: "https://myweb.site" 
+    security_contact: "my-validator0@gmail.com"
+  status: 1 
+  tokens: "100"
+```
+
+Usually when first creating a validator, the immediate status will be 
+`BOND_STATUS_UNBONDED`. To see your validator's status change you will need to 
+wait for the epoch to end.
+
+### 5.2 Understanding Validator Status
+
+Your validator enters the active set based on two conditions: 
+1. Having sufficient stake to qualify for the active set.
+2. The completion of the epoch (a network-wide time period for 
+coordinating activities) in which your validator qualified for the active set.
+
+When active, your status will show as `BOND_STATUS_BONDED`.
+
+The other status codes are:
+
+```shell
+BOND_STATUS_UNSPECIFIED = 0
+BOND_STATUS_UNBONDED = 1
+BOND_STATUS_UNBONDING = 2
+BOND_STATUS_BONDED = 3
+```
+
+### 5.3 Managing Your Validator
+
+> ⚠️ **Important**: Babylon uses the `epoching` module to wrap staking 
+> operations. 
+> All staking-related transactions (delegate, redelegate, unbond) must be 
+> executed through the `x/epoching` module, the `x/epoching` module wraps the 
+> commands of `x/staking` and all staking related transactions will take 
+> into effect at the end of the epoch.
+
+For validator operations, use the standard Cosmos SDK staking module commands:
```

Comment:
It's a little bit confusing at `use the standard Cosmos SDK staking module commands`.
Aren't validators expected to use `x/epoching` module` commands?